src/actions.js

```diff
@@ -305,7 +305,23 @@ exports.getActions = function() {
 			label: 'VideoCall - Select Audio Source',
 			options: [
 				input, 
-				audioBusMaster
+				{
```

Comment:
This was labelled as a typo fix, however @wojtek14a what is this change to the Bus input?

Reply:
that seems to be an inclution of the correct set of Bus options. I seem to have missed that it also used the headphones as an option, and therefore needs it's owned list. Well spottet